src/main/java/com/redhat/cloud/notifications/routers/ApplicationService.java

```diff
@@ -29,14 +29,14 @@
     ApplicationResources appResources;
 
     @GET
-    public Multi<Response> getApplications(@QueryParam("bundleName") String bundleName) {
+    public Uni<Response> getApplications(@QueryParam("bundleName") String bundleName) {
         // Return configured with types?
         if (bundleName == null || bundleName.isBlank()) {
-            return Multi.createFrom().failure(new IllegalArgumentException("there is no bundle name given. Try ?bundleName=xxx"));
+            return Uni.createFrom().failure(new IllegalArgumentException("there is no bundle name given. Try ?bundleName=xxx"));
         }
         // TODO how can we find out there is nothing to send and return a 404 ?
-        return appResources.getApplications(bundleName).onItem()
-                .transform(app -> { return Response.ok(app).build(); });
+        return appResources.getApplications(bundleName)
+                .collectItems().asList().onItem().transform(apps -> Response.ok(apps).build());
```

Comment:
Returning a `Uni<List<Application>>` in this method should produce the same output:
```suggestion
        return appResources.getApplications(bundleName).collectItems().asList();
```

Reply:
Ah yes, that would be better and I think the openapi would get generated more natural this way.